src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/CatchParameterNameCheck.java

```diff
@@ -52,6 +52,37 @@
  * <pre>
  * &lt;module name="CatchParameterName"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class MyTest {
+ *   public void myTest() {
+ *     try {
+ *       // ...
+ *     } catch (ArithmeticException e) { // OK
+ *       // ...
+ *     } catch (ArrayIndexOutOfBoundsException ex) { // OK
+ *       // ...
+ *     } catch (IOException t) { // OK
```

Comment:
`IOException` please change to Throwable, t is more used with Throwable

Reply:
done